docs/docs-ref-autogen/excel/excelscript/global.officescript.emailattachment.yml

```diff
@@ -0,0 +1,40 @@
+### YamlMime:TSType
+name: Global.OfficeScript.EmailAttachment
+uid: 'ExcelScript!Global.OfficeScript.EmailAttachment:interface'
+package: ExcelScript!
+fullName: Global.OfficeScript.EmailAttachment
+summary: >-
+  The attachment to send with the email. A value must be specified for at least one of the `to`<!-- -->, `cc`<!-- -->,
+  or `bcc` parameters. If no recipient is specified, the following error is shown: "The message has no recipient. Please
+  enter a value for at least one of the "to", "cc", or "bcc" parameters."
+remarks: ''
+isPreview: true

```

Comment:
@AlexJerabek Could you take another look at this PR? 

I added the preview APIs, but it looks like there's no preview tagging in intellisense-preview.d.ts, and no preview tagging on the preview APIs, so I manually added @beta tags to the preview APIs. Here's what it looks like: https://review.learn.microsoft.com/en-us/javascript/api/office-scripts/excelscript/global.officescript.emailattachment?view=office-scripts&branch=pr-en-us-380

Note that the preview APIs are also adding the new "global" namespace.